build.gradle.kts

```diff
@@ -84,6 +84,7 @@ dependencies {
     implementation("org.apache.commons:commons-csv:1.3")
     implementation("commons-cli:commons-cli:1.4")
 
+    api(log4j("core"))
```

Comment:
Since this is becoming a part of API I think we need a `CHANGELOG` entry for this change too.
> - Add `log4j-core` to the API.

Reply:
Sure